pkgs/applications/misc/qsstv/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, qtbase, qmake, openjpeg, pkgconfig, fftw, libpulseaudio,
+  alsaLib, hamlib, libv4l, fftwFloat }:
+
+stdenv.mkDerivation rec {
+  version = "9.2.6";
+  name = "qsstv-${version}";
+
+  src = fetchurl {
+    url = "http://users.telenet.be/on4qz/qsstv/downloads/qsstv_${version}.tar.gz";
+    sha256 = "0sx70yk389fq5djvjwnam6ics5knmg9b5x608bk2sjbfxkila108";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [
+    qmake
+    pkgconfig
+  ];
+
+  buildInputs = [ qtbase openjpeg fftw libpulseaudio alsaLib hamlib libv4l
+                  fftwFloat ];
+
+  installPhase = ''
+    make INSTALL_ROOT=$out install
+    install -D ${./qsstv.desktop} $out/share/applications/qsstv.desktop
+    install -D qsstv/icons/qsstv.png $out/usr/share/pixmaps/qsstv.png
+    substituteAll ${./qsstv.desktop} $out/share/applications/qsstv.desktop
```

Comment:
Usually we use `makeDesktopItem` for new packages.

```nix
desktopItem = makeDesktopItem {
  name = "eduke32";
  exec = "eduke32-wrapper";
  comment = "Duke Nukem 3D port";
  desktopName = "Enhanced Duke Nukem 3D";
  genericName = "Duke Nukem 3D port";
  categories = "Application;Game;";
};
```

See also pkgs/games/eduke32/default.nix

Reply:
Thanks for this information. I changed the files accordingly.